misc: revert 'Level Up Twilio At Scale' webinar changes

Hunga1 opened the pull request:
Reverted all changes associated with the 'Level Up - Integrating Twilio At Scale with Go' webinar sample application. State of the repo now reflects the base demo application pre-webinar (commit: e00608f61ed05e53397f2edb0dc0f9e301bc1974)

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

Hunga1 committed:
Revert "fix: prometheus settings (#3)"

This reverts commit 6fa13339288d37cc0d595013ad4786d135373fac.

Hunga1 committed:
Revert "adding docker and make updates"

This reverts commit c5c8d080263ba2250b0f89c21c3ce689258ea76b.

Hunga1 committed:
Revert "Update README with the env variables"

This reverts commit c5d7cbf2628bc24de4e72ad0360b024ef25a0d18.

Hunga1 committed:
Revert "Addinig initial Prometheus"

This reverts commit 22a1e6e0045ce42c2c18ff3912ff64a58348d7b5.

Hunga1 commented:
Marked the PR as a draft until we are ready to convert the main branch to it's pre-webinar state. When we are ready, the draft status can simply be removed, and this is ready to be reviewed/merged.

Hunga1 commented:
All changes made to this project for the webinar have been retained in a separate branch: [level_up_twilio_at_scale_webinar](https://github.com/twilio-labs/sample-twilio-go/tree/level_up_twilio_at_scale_webinar)

rakatyal commented on README.md:
```diff
@@ -136,30 +109,30 @@ For Twilio Go SDK documentation, examples, and code snippets, please refer to th
+- Initializing the Twilio SDK client. A pointer to this client instance is then saved in each application's service `client` field for later use (i.e. [SMSService.client](https://code.hq.twilio.com/twilio/go-review-rewards-example-app/blob/main/pkg/sms/sms_service.go#L27))
 
-    https://github.com/twilio-labs/sample-twilio-go/blob/main/cmd/app/main.go#L115
+    https://code.hq.twilio.com/twilio/go-review-rewards-example-app/blob/main/main.go#L44
```
Can we change these links to GH ones?

rakatyal commented on README.md:
```diff
@@ -1,36 +1,30 @@
 ## Build the application
 
-To build the application, run `make build-app` to build the application binary in `out/bin/sample-twilio-go`
+To build the application, run `go build`
```
Do we need to specify the output binary name here?

Hunga1 replied:
Changed the build instructions so that we explicitly build a binary named `sample-twilio-go` into a `out/bin` child directory. Instructions to run the application reflect the change as well.

rakatyal commented on README.md:
````diff
@@ -70,35 +64,29 @@ https://1404-2601-282-1200-118-e1d0-ba12-d474-d43a.ngrok.io/sms
 
 ```
-make docker-compose
+./review-rewards-example-app -from=+15555555555 -url=https://1404-2601-282-1200-118-e1d0-ba12-d474-d43a.ngrok.io
````
Is this the output binary? Not sure if it will run from the root folder?

rakatyal commented on go.mod:
```diff
@@ -1,63 +1,35 @@
-module github.com/twilio-labs/sample-twilio-go
+module code.hq.twilio.com/twilio/review-rewards-example-app
```
We should change this to GH as well.

Hunga1 replied:
Changed the module name to the new repo location; `github.com/twilio-labs/sample-twilio-go`

rakatyal commented on pkg/controller/review_controller.go:
```diff
@@ -1,76 +1,38 @@
-	"sync"
 	"time"
 
+	"code.hq.twilio.com/twilio/review-rewards-example-app/pkg/sms"
```
Here too.

Hunga1 committed:
Updated leftover references to old repo/module name

Updated leftover references to old repo. Updated README with new instructions for building and running the app.
